spec/inquisition/collector_spec.rb

```diff
@@ -14,10 +14,39 @@
           def call
             ['a', ['b', 'c']]
           end
+
+          def self.enabled?
+            true
+          end
         end
       end
 
       it { is_expected.to match_array(['a', 'b', 'c']) }
     end
+
+    context 'call enabled runner' do
+      subject(:collector) { described_class.new(collection: collection).call }
```

Comment:
In general, I think would be better to create something similar to the previous test with only 1 difference:
we expect that this runner will not receive `.new`